composer.json

```diff
@@ -1,16 +1,16 @@
 {
   "require": {
+    "php": ">=5.4.0",
```

Comment:
We should require PHP >= 5.6.0 since we use `use function` construct in our custom Behat contexts.